### DISCUSSION TITLE
DAI 2023-2024 - Practical work 1 - Guillaume Trüeb and Gwendal Piemontesi

### DISCUSSION THREAD

**Discussion by grinlemon** [heig-vd-dai-course/heig-vd-dai-course#63] [Show and tell]
We will be working together! :)

**Comment by hadrylouis** [comment]
Could you please tell us a bit about what you plan to do for this practical work? :)

**Reply by grinlemon** [reply]
It's top secret 🕵🏻‍♂️ @hadrylouis

**Comment by grinlemon** [comment]
We would like to try something a bit crazy. We would like to take a song (probably .mp3) and read it's bytes. With it we would like to create an art picture! @hadrylouis

**Reply by ludelafo** [reply]
Perfect!

As discussed yesterday, start small and add more features if you have time.

My advice would be to split the project in three phases:

1. Create an artwork from a seed
2. Use the MP3 to set the seed for the artwork
3. Add parameters to tweak the seed/the MP3 file/the artwork to give the user some flexibility to the generation.

**Comment by grinlemon** [comment]
After a lot of effort we have finished our first practical work!
Link to the repo : https://github.com/truebguillaume/Kuby
Hash of last commit : a3d1a38766b9635f51c97f53cd9b9646e4edcafb
